release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -44,6 +44,14 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-12-secure-execution-z-linux-one"]
+==== IBM Secure Execution on {ibmzProductName} and LinuxONE (Technology Preview)
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and LinuxONE (s390x architecture) as a Technology Preview feature. IBM Secure Execution provides the highest level of isolation and security for cluster workloads, and you can enable it by using a IBM Secure Execution-ready QCOW2 boot image.
```

Comment:
Can we mention that IBM Secure Execution is a Hardware Enhancement and protects Memory Boundaries for KVM Guests?

Reply:
For sure, added